ci/install-python-dependencies.sh

```diff
@@ -4,3 +4,4 @@ pip install --upgrade pip
 pip install --user pyhocon
 pip3 install --user pyhocon
 pip install --user pep8
```

Comment:
`pip install --user pep8` could be removed, right?

Reply:
yeah, it can be removed. Will remove it. Was looking out for dependency failures if any.

Reply:
Seems like the tests are failing after the change:
```
job-server-python/src/python/test/apitests.py:149:1: E305 expected 2 blank lines after class or function definition, found 1
```